### ISSUE TITLE
Filter by max score with GDPR export

### ISSUE THREAD

**Issue by skorokithakis** [andrewbanchich/shreddit#83]
Is it possible to filter by max score with the GDPR export? Can the client check the score locally, if you're already fetching the comment before deleting?

**Comment by andrewbanchich** [comment]
It is definitely possible, and should be fairly easy to add if it doesn't already work. I assume you've tried and it didn't work?

**Comment by skorokithakis** [comment]
Yes, it complained that max score is not compatible with GDPR export. Oddly, it didn't exit with an error, but it complained about the same thing for every single item in the export (and it didn't try to delete them, so it kept printing screenful after screenful of text).

**Comment by andrewbanchich** [comment]
Ah I see. Yeah, this should be fairly easy to do, but unfortunately I'm pretty busy and probably won't have time to look at it any time soon.

PRs are always welcome though, and I don't mind assisting with any questions.

**Comment by skorokithakis** [comment]
My Rust is... rusty, but I can give it a go!

Also, I've used this for simple issues, and it was pretty useful, if you want to try it:

https://sweep.dev/
